src/Events/Event.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyperdrive\Events;
+
+use Hyperdrive\Combat\Combat;
+use Hyperdrive\Combat\Enemies;
+use Hyperdrive\Geography\PlanetSurface;
+use Hyperdrive\HyperdriveNavigator;
+use Hyperdrive\Output\OutputContract;
+use Hyperdrive\Pilot\Pilot;
+use Hyperdrive\Quest\Quest;
+use Hyperdrive\Quest\QuestLog;
+use Hyperdrive\Ship\Ship;
+
+class Event
+{
+    protected OutputContract $output;
+
+    public function __construct(OutputContract $output)
+    {
+        $this->output = $output;
+    }
+
+
+    public function randomSpaceEvents(Pilot $player, Ship $playerShip, HyperdriveNavigator $hyperdrive, QuestLog $questlog, PlanetSurface $surface, Event $event): void
+    {
+        $random = rand(1, 10);
+
+        if ($random == 8) {
```

Comment:
Czasem używasz porówynania "==", a czasem "===". Myślę, że warto trzymać się "===". Gdybyś odpałał czasem ECS, to zwracałby Ci na to uwagę.

Reply:
Dopóki nie dostałem komentarzy o ECS, nie wiedziałem, że coś takiego nawet istnieje.